src/utils.js

```diff
@@ -8,6 +8,20 @@ import Image from 'components/Image';
 import Anchor from 'components/Anchor';
 
 
+export const getFormattedDateString = (dateString) => {
+  // Format an date string to our preferred format
+  // Expects YYYY-MM-DD and converts to DD-MM-YYYY
+  return dateString.split('-').reverse().join().replaceAll(',', '-');
+};
+
+
+export const getFormattedTimeString = (dateTimeString) => {
+  // Format an ISO date string to our preferred format
+  // Eg. 2021-09-20T08:00:00+02:00 to 08:00
+  return dateTimeString.split('T')[1].split(':').slice(0, 2).join().replace(',', ':');
+};
```

Comment:
This is bonkers - in my Brave browser (Chromium based) I can get an actual `Date` instance like this:

```js
new Date('2021-09-09T17:25:00+02:00')
```

This should then be fed to react-intl components or underlying imperative API and be properly formatted given the locale from the browser.

Reply:
Ya we can do this.  I am a little worried about the performance impact of this since it's possible there will be a lot of dates or times but for now I suppose we can convert the string to a date and then use react-intl components.